bin/growpart

```diff
@@ -375,6 +385,14 @@ resize_sfdisk() {
 				"Additional space will go unused."
 	fi
 
+	if [ -n "${free_percent}" ]; then
+		# Shrink max_end to keep X% of whole disk unused
+		# (for overprovisioning)
+		free_percent_sectors=$((sector_num/100*free_percent))
+		debug 1 "reserving ${free_percent_sectors} sectors (${free_percent}%) for overprovisioning"
+		max_end=$((max_end-free_percent_sectors))
```

Comment:
Worth checking that `max_end` is positive?

Reply:
Yupe, working on that currently

Reply:
I have rewritten the overprovisioning code and so I don't think this is now applicable.